pontoon/contributors/utils.py

```diff
@@ -104,6 +104,14 @@ def users_with_translations_counts(
     # Assign properties to user objects.
     contributors = User.objects.filter(pk__in=user_stats.keys())
 
+    # Exclude system users
+    system_users_emails = [
+        "pontoon-sync@mozilla.com",
+        "pontoon-gt@mozilla.com",
+        "pontoon-tm@mozilla.com",
+    ]
```

Comment:
We actually changed these email address to [@example.com](https://github.com/mozilla/pontoon/blob/cc8fb121e1cd5663f447de5713d971f7d44e34f0/pontoon/base/migrations/0009_change_pontoon_users_emails.py).

However, it'd be nicer if we had a [`UserProfile.system_user`](https://github.com/mozilla/pontoon/blob/cc8fb121e1cd5663f447de5713d971f7d44e34f0/pontoon/base/models.py#L1649) flag, which would allow us to reference these users more easily and make the code more future-proof.

Additionally, we should make use of such flag in at least two other places:

https://github.com/mozilla/pontoon/blob/cc8fb121e1cd5663f447de5713d971f7d44e34f0/pontoon/insights/tasks.py#L118

https://github.com/mozilla/pontoon/blob/cc8fb121e1cd5663f447de5713d971f7d44e34f0/pontoon/base/views.py#L680